### ISSUE TITLE
unrecognized arguments error

### ISSUE THREAD

**Issue by dba91** [MouseLand/suite2p#645]
Hello, I have an older version of s2p running smoothly on one machine, but recently installed a newer s2p on a second PC. When I try to process a file (which processes just fine on the first machine), it's immediately interrupted by a "unrecognized arguments" error that points to the ops and db files (no db file is yet created though, by the time the error occurs). I originally tried with custom ops settings, but the same error occurs with the default. I did not see this problem described in past comments.
Here's the error with the default ops:
![image](https://user-images.githubusercontent.com/31355751/112355915-5afffa00-8c8b-11eb-8a43-21b1d3615c6d.png)
![image](https://user-images.githubusercontent.com/31355751/112356018-710dba80-8c8b-11eb-8566-43b7ea13bba5.png)

**Comment by dba91** [comment]
Here is the content of the setup file in case it is of use:

**Comment by gongzhongyan** [comment]
Hi, I had the same problem. When I run `(suite2p) C:\Users\xxx Lab\suite2p-main>suite2p` in command line, the suite2p was successfully activated. But when I ran my data, I had the same `unrecognized arguments error`.
<img width="480" alt="Screen Shot 2021-03-24 at 3 48 01 PM" src="https://user-images.githubusercontent.com/63259578/112393455-5271e880-8cb8-11eb-9646-0cc992cb1b3a.png">

**Comment by gongzhongyan** [comment]
Hi, I forced installing an older version and solved the problem. I used `pip install suite2p==0.9.3`

**Comment by dba91** [comment]
That worked for me as well, thank you for posting!

**Comment by dba91** [comment]
Was this bug tackled? I recently upgraded to access cellpose and ran into this error again.

**Comment by dba91** [comment]
Carsen, I finally realized this is due to the space in my Windows Path Username thanks to your comment. I solved this for myself by just creating a new user account and trying there. Worked for me! Apart from all the modules installed with the environment.yml file, I had to to separately pip install a few things like torch, cellpose, and downgrade the PyQt5 (was getting this error: AttributeError: module 'PyQt5.QtGui' has no attribute 'QPushbutton').
